modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/info/item/provider/JournalArticleInfoItemFieldValuesProvider.java

```diff
@@ -239,6 +239,11 @@ private List<InfoFieldValue<Object>> _getJournalArticleInfoFieldValues(
 					themeDisplay);
 
 				if (Validator.isNotNull(articleImageURL)) {
+					journalArticleFieldValues.add(
```

Comment:
@cgoncas, can't we use the existing field smallImageInfoField for now and not create a new one? I know that we want to have standardized fields in the future, and we would need to adapt it later on. What do you think?

Reply:
Hey @jkappler , we have added this new field to blogs and documents and media, only standardized names are being used in the contend dashboard, for now.

We did the same with publish date, instead display date.